addons/test_mail/tests/test_message_post.py

```diff
@@ -290,3 +291,7 @@ def test_post_post_w_template(self):
             subject='About %s' % test_record.name,
             body_content=test_record.name,
             attachments=[('first.txt', b'My first attachment', 'text/plain'), ('second.txt', b'My second attachment', 'text/plain')])
+
+    def test_message_post_return(self):
+        messageId = call_kw(self.env['mail.channel'], 'message_post', [1], {'body': 'test'})
+        self.assertTrue(isinstance(messageId, int))
```

Comment:
try to run the test without your fix see it it fails.

Also I would add a bit more comments here that the goal is to simulate rpc return specifically, but it is done this way for simplicity.